CHANGELOG.md

```diff
@@ -83,8 +83,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Remove helper functions and other code for handling OTP [#705](https://github.com/greenbone/gvmd/pull/705) [#709](https://github.com/greenbone/gvmd/pull/709) [#713](https://github.com/greenbone/gvmd/pull/713) [#735](https://github.com/greenbone/gvmd/pull/735) [#748](https://github.com/greenbone/gvmd/pull/748) [#749](https://github.com/greenbone/gvmd/pull/749)
 - Remove stray prototype nvt_iterator_copyright [#721](https://github.com/greenbone/gvmd/pull/721)
 
-
-[Unreleased]: https://github.com/greenbone/openvas/compare/gvmd-8.0...master
+[9.0.0]: https://github.com/greenbone/openvas/compare/gvmd-8.0...v9.0.0
 
 ## [8.0.1] (unreleased)
```

Comment:
I wasn't sure if all the changes from the 8.0.1 release were also included in the branch, but I've checked and the changelog sections for the 9.0 and the 8.0 one are basically identical, so it should be okay to use the released version as the base:
```suggestion
## [8.0.1] (2019-07-17)
```